src/convenience/constants.ts

```diff
@@ -18,11 +18,34 @@ const ALL_CHAT_PERMISSIONS = {
     can_manage_topics: true,
 } as const;
 
+/**
+ * Contains utility types for the available update types and chat permissions
+ */
+export interface ApiConstants {
+    DefaultUpdateTypes: typeof DEFAULT_UPDATE_TYPES[number];
+    AllUpdateTypes: typeof ALL_UPDATE_TYPES[number];
+    AllChatPermissions: keyof typeof ALL_CHAT_PERMISSIONS;
```

Comment:
I don’t see this as being more useful than exporting only the keys, since they’re all `true` anyway.
Wouldn’t it be better for us to export the keys and allow the user to do `Record<AllChatPermissions, boolean>` if they need an object?

Reply:
The question is whether there is any reason to have this type at all.

Reply:
I figured since we're exporting types for constants, might as well export this one.
As for having only the keys, it's consistent with the other two types, and with the information we're conveying, which is: "what are all possible chat permissions?" "a: this, this, and that."